### PR TITLE
Fix wrapping commas on header links

### PR DESCRIPTION
**Pull request by jonalport**
Tried to do this with pseudo-elements but after much difficulty decided to stick to basics.

### CODE REVIEW

File: app/views/categories/_top_links.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <ul class="category-link-list">
   <% category.links.each do |link| %>
     <li class="category-link-list__item">
-      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %></a>
+      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %><%= ',' unless link === category.links.last %></a>
```
Comment (asmega) [inline app/views/categories/_top_links.html.erb]: 2 things. should the comma have a space after it like the css? and is `==` not sufficient as the equality check?

Comment (jonalport) [inline app/views/categories/_top_links.html.erb]: As far as the comma after the space, I intentionally left that out as the spacing is added more explicitly via CSS margin.

What I was trying to do with the === was an object comparison (to see if they were the same object), as opposed to a value comparison. It seems like that is not really a thing, however == does not do it either. So I have settled on ".equals?" which works. Sound good?

**Comment by asmega** [comment]
:+1:

File: app/views/categories/_top_links.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <ul class="category-link-list">
   <% category.links.each do |link| %>
     <li class="category-link-list__item">
-      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %></a>
+      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %><%= ',' unless link.equal? category.links.last %></a>
```
Comment (benbarnett) [inline app/views/categories/_top_links.html.erb]: Surprised there's not a more succinct way to do that in ruby. Doesn't seem there is though!

**Comment by benbarnett** [comment]
:+1:

File: app/views/categories/_top_links.html.erb (hunk 2 above)
Comment (jonalport) [inline app/views/categories/_top_links.html.erb]: Agreed!

**Pull request merged**